### PR TITLE
Covering another part of Meteor API

### PR DESCRIPTION
**Pull request by alesn**

### CODE REVIEW

**Comment by radekmie** [comment]
Hello @alesn! Well, you've done a lot of changes there! I'd love to add these changes, but you've also changed the whole codebase style... Let's split it into two parts: at first, add the changes and then, eventually, add the code style config. I'm actually using one right now but it's not in the repo - I know, my fault. If you want, I can add it and we'll manage it further.

**Comment by alesn** [comment]
Yes, my files are formatted by Prettier source code formatter that also Facebook adopted and uses. Check it out if you don't use it yet. I don't deal with "codebase style" any more. That is handled fully by Prettier in a consensus-based standard way and the actual source code remains as what matters

**Comment by radekmie** [comment]
I know Prettier, really. It's more about having the complete config of it (including its version) in the repo, not in your editor - otherwise, it's kind of useless.

**Comment by alesn** [comment]
Sure, if you want to reformat the source code in a different way for your repository, you're naturally free to do so. I contributed the declarations that I needed and could be useful for others

**Comment by radekmie** [comment]
Yeah, but you've also changed the whole codebase... Please, either make a PR with only meaningful changes or add the configs.

**Comment by alesn** [comment]
I've added .prettierrc

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "arrowParens": "always",
+  "trailingComma": "es5",
+  "printWidth": 100
+}
```
Comment (radekmie) [inline .prettierrc]: I'd go with this config, as I use it everywhere else:

```json
{
  "bracketSpacing": false,
  "singleQuote": true
}
```

If you don't have time, I can do it. Also, add the trailing newline, please.

**Review by radekmie — CHANGES_REQUESTED** [review]
I'll set up a CI for checking the code style later.

File: .prettierrc (hunk 1 above)
Comment (alesn) [reply .prettierrc]: I don't mind what formatting you like to use here. You can format them any way you like (I won't have time for that). Anyway, these declarations should be contributed to the official flow-typed repo eventually. Preparing the final version for contribution there I think the best bet would be to format the source code by Prettier with no explicit configuration and going with its defaults ... However, flow-typed repo doesn't have any set code style as far as I know (the declarations in a readable style is what matters)

Comment (radekmie) [reply .prettierrc]: So I'll change and reformat it then. Thanks!

**Review by radekmie — APPROVED** [review]

**Pull request merged**